src/routes/conversation/[id]/+server.ts

```diff
@@ -128,6 +143,16 @@ export async function POST({ request, fetch, locals, params }) {
 				},
 			}
 		);
+
+		// if messages API rate is limited, save message event
+		if (locals.user && typeof MESSAGES_RATE_LIMIT !== "undefined") {
```

Comment:
```suggestion
		if (locals.user && MESSAGES_RATE_LIMIT) {
```

It won't be undefined, it'll be empty string at most I think